My Simple App/Application/WeatherViewModel.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  ViewModel.swift
+//  My Simple App
+//
+//  Created by Erick Sanchez on 8/5/20.
+//  Copyright © 2020 Erick Sanchez. All rights reserved.
+//
+
+enum AddressResult {
+  case validAddress(Address)
+  case invalidAddress(Address)
+  case error(Error)
+}
+
+class WeatherViewModel {
+  private(set) var validAddresses = [Address]()
+  private(set) var invalidAddresses = [Address]()
```

Comment:
woodward4422 2 days ago @woodward4422
Why are these two seperate arrays? Why do we need to store invalid adresses in a variable?

LinnierGames 1 hour ago Author Owner
IIUC, having two variables reads better than a 2-dimensional array named something like var addresses: [[Address]]() where addresses[0] contains the list of valid addresses and addresses[1] contains the list of invalid addresses